<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="lbTitle">
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Welcome to online reservation plateform</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidgetAPP">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Manage flight</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Search flight :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbDeparture">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QPushButton" name="pbSearch">
          <property name="styleSheet">
           <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="cbArrival">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pbConfirmeAndBook">
            <property name="styleSheet">
             <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhExclusiveInputMask</set>
            </property>
            <property name="text">
             <string>Confirme and book </string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbNewResearch">
            <property name="styleSheet">
             <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>New Research</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="7">
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>28</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Departure</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Arrival</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dep time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Arri time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Flight_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Route_id</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Departure :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Arrival :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Departure time :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QDateTimeEdit" name="dateTimeEdit"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Personnal information:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>First name : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leFirstName"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Last name : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leLastName"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Date of Birth :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateBirth"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Phone number :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePhoneNumber"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Email :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leEmail"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbValidate">
            <property name="styleSheet">
             <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Validate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReservation">
       <attribute name="title">
        <string>Manage Reservations</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="9">
         <widget class="QTableWidget" name="tableWReservation_">
          <column>
           <property name="text">
            <string>Reservation n°</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Firstname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Lastname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Departure time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Departure airport</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Arrival airport</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Route n°</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lb_Firstname_res">
          <property name="text">
           <string>Firstname :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lb_Lastname">
          <property name="text">
           <string>Lastname :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="lbEmail_res">
          <property name="text">
           <string>Email :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leFirstname_res"/>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="leLastname_res"/>
        </item>
        <item row="2" column="6">
         <widget class="QPushButton" name="pbTicket_res">
          <property name="text">
           <string>Print ticket</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="leLastname_res_2"/>
        </item>
        <item row="0" column="8">
         <widget class="QPushButton" name="pbSearchRes">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QPushButton" name="pbReset_res">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
